.github/workflows/toolchain-version-check.yaml

```diff
@@ -0,0 +1,76 @@
+---
+name: Toolchain Version Check
+"on":
+  schedule:
+    - cron: "0 5 * * 5" # 5am UTC Fridays
+
+jobs:
+  check-toolchain-version:
+    name: Check toolchain version
```

Comment:
Name of file = "toolchain version check"
name of the job = "check toolchain version".

Should we unify these? Happy either way, just pointing it out.

Reply:
Doesn't bother me either way, up to you. My only ask is that if this workflow ends up appearing in multiple repos is that we do it the same.